source/llassetgen/include/llassetgen/Atlas.h

```diff
@@ -29,10 +29,11 @@ namespace llassetgen {
         Image atlas{packing.atlasSize.x, packing.atlasSize.y, bitDepth};
         atlas.clear();
 
-        auto rectIt = packing.rects.begin();
-        for (; imgBegin < imgEnd; rectIt++, imgBegin++) {
-            Image view = atlas.view(rectIt->position, rectIt->position + rectIt->size);
-            view.copyDataFrom(*imgBegin);
+#pragma omp parallel for
```

Comment:
i prefer same indent as next line

Reply:
We use a formatting tool which keeps putting that indent as it is right now. I can understand your preference though.